Binary files orig/msa/src/gc-7.2/.msaMakefile.win.swp and patched/msa/src/gc-7.2/.msaMakefile.win.swp differ
diff -Nru orig/msa/src/gc-7.2/msaMakefile.win patched/msa/src/gc-7.2/msaMakefile.win
--- orig/msa/src/gc-7.2/msaMakefile.win	2020-10-27 10:48:51.000000000 -0700
+++ patched/msa/src/gc-7.2/msaMakefile.win	2021-06-25 14:06:40.653560500 -0700
@@ -9,8 +9,8 @@
 
 gc72:
 	export PKG_LIBS="$(PKG_LIBS) -I$(AO_INCLUDE_DIR) -I$(INCLUDE_DIR) -I$(PRIVATE_INCLUDE_DIR)"; \
-	export PKG_CFLAGS="$(PKG_CFLAGS) -DALL_INTERIOR_POINTERS -DGC_DLL -DGC_THREADS -D_CRT_SECURE_NO_DEPRECATE -I$(AO_INCLUDE_DIR) -I$(INCLUDE_DIR) -I$(PRIVATE_INCLUDE_DIR)"; \
-	export PKG_CXXFLAGS="$(PKG_CXXFLAGS) -DALL_INTERIOR_POINTERS -DALL_INTERIOR_POINTERS -DGC_DLL -DGC_THREADS -D_CRT_SECURE_NO_DEPRECATE -I$(AO_INCLUDE_DIR) -I$(INCLUDE_DIR) -I$(PRIVATE_INCLUDE_DIR)"; \
+	export PKG_CFLAGS="-fcommon $(PKG_CFLAGS) -DALL_INTERIOR_POINTERS -DGC_DLL -DGC_THREADS -D_CRT_SECURE_NO_DEPRECATE -I$(AO_INCLUDE_DIR) -I$(INCLUDE_DIR) -I$(PRIVATE_INCLUDE_DIR)"; \
+	export PKG_CXXFLAGS="-fcommon $(PKG_CXXFLAGS) -DALL_INTERIOR_POINTERS -DALL_INTERIOR_POINTERS -DGC_DLL -DGC_THREADS -D_CRT_SECURE_NO_DEPRECATE -I$(AO_INCLUDE_DIR) -I$(INCLUDE_DIR) -I$(PRIVATE_INCLUDE_DIR)"; \
 	${R_HOME}/bin${R_ARCH_BIN}/R.exe CMD SHLIB -o gc72.dll *.c; \
 	${R_HOME}/bin${R_ARCH_BIN}/R.exe CMD SHLIB -o gccpp72.dll -L. -lgc72 *.cpp; \
 	$(AR) rcs libGC.a $(OBJNamesGC); \
